setup.py

```diff
@@ -141,7 +141,10 @@ def pkgconfig(*packages, **kw):
     platform = 'ios'
 if exists('/opt/vc/include/bcm_host.h'):
     platform = 'rpi'
-if exists('/usr/lib/arm-linux-gnueabihf/libMali.so'):
+if(
+    exists('/usr/lib/arm-linux-gnueabihf/libMali.so')
+    or exists('/usr/local/mali-egi/libmali.so')
```

Comment:
Shouldn't this path be `/usr/local/mali-egl/libmali.so` (instead of `mali-egi`)? Also, Arch Linux ARM installs this library to a different path. Would it make sense to allow the path to the libGL to be specified with an environment variable during the build?